stumpy/aampi.py

```diff
@@ -35,6 +35,13 @@ class aampi:
         Note that this will increase the total computational time and memory usage
         when k > 1.
 
+    mp : numpy.ndarry, default None
+        A 2D array of shape `(len(T) - m + 1, 2 * k + 2)`, where the first k columns
```

Comment:
You've jumped right into the description but you should really start off with "A pre-computed matrix profile (and corresponding matrix profile indices). This is a 2D array..."

Reply:
Great point! Thanks for the input. I will revise the docstrings.